Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -28,6 +54,40 @@ def edit_args
     end
   end
 
+  sig { params(path: Pathname, cask: T::Boolean).void }
+  def fail_with_message(path, cask)
```

Comment:
```suggestion
  def raise_unless_existing!(path, cask)
```

Reply:
@abitrolly Still prefer this naming or at least something with `raise` and the `!` to be more in keeping with typical Ruby behaviour.

Reply:
Renamed to `raise_with_message!`